otx/api/usecases/exportable_code/demo/requirements.txt

```diff
@@ -1,3 +1,2 @@
-openvino==2022.1.0
```

Comment:
The otx demo currently uses that requirement. Since nncf, openvino, and otx are referencing out-of-date versions, problems may occur in the demo after the version update.

I've modified this to a working version, but it seems need code to automatically update it to always have the latest requirements. (I don't have an idea right now, so I won't proceed with the PR.)